flink-addons/flink-gelly/src/main/java/org/apache/flink/gelly/Edge.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.gelly;
+
+import java.io.Serializable;
+
+import org.apache.flink.api.java.tuple.Tuple3;
+
+public class Edge<K extends Comparable<K> & Serializable, V extends Serializable> 
```

Comment:
This Edge class, and all the 3 other Edge* classes below are missing javadocs. The rest of the graph api is very well documented, but these classes not.